### PR TITLE
Config added for earlier/later feature

### PR DESCRIPTION
**Pull request by ValentinCousien**

### CODE REVIEW

File: Cartfile [hunk 1]
```diff
@@ -1 +1 @@
-github "CanalTP/NavitiaSDKUX_ios" == 2.1.0
+github "CanalTP/NavitiaSDKUX_ios" "develop"
```
Comment (RachikAbidi) [inline Cartfile]: You need to specify the version instead of the branch name

File: plugin.xml [hunk 2]
```diff
@@ -14,7 +14,7 @@
             </feature>
         </config-file>
         <source-file src="src/ios/CDVNavitiaSDKUI.swift" />
-        <framework src="Carthage/Build/iOS/NavitiaSDKUI.framework" custom="true" embed="true" />
+        <framework src="NavitiaSDKUI.framework" custom="true" embed="true" />
```
Comment (RachikAbidi) [inline plugin.xml]: Please rollback this change

File: src/ios/CDVNavitiaSDKUI.swift [hunk 3]
```diff
@@ -31,6 +31,7 @@ import NavitiaSDKUI
         let destinationColor = toUIColor(hexColor: config["destinationColor"] as? String) ?? UIColor(red:0.69, green:0.01, blue:0.33, alpha:1.0)
         let multiNetwork = config["multiNetwork"] as? Bool ?? false
         let formJourney = config["formJourney"] as? Bool ?? false
+        let isEarlierLaterFeatureEnable = config["isEarlierLaterFeatureEnable"] as? Bool ?? false
```
Comment (RachikAbidi) [inline src/ios/CDVNavitiaSDKUI.swift]: Can we have a better name for this feature parameter ? And anyway, `isEarlierLaterFeatureEnable` should be `isEarlierLaterFeatureEnable**d**` or `enableEarlierLaterFeature`

Comment (nicolas-chaix) [reply src/ios/CDVNavitiaSDKUI.swift]: done

File: Cartfile (hunk 1 above)
Comment (hirsch-adeline) [reply Cartfile]: The version doesn't exist yet

**Pull request merged**